ee/fe/modules/500-openvpn/openapi/config-values.yaml

```diff
@@ -58,9 +58,13 @@ properties:
   tunnelNetwork:
```

Comment:
Due to we have ovpn installations exclusively in FE, it makes sense to make migration from `tunnelNetwork` to `tcpTunnelNetwork`.